ferrocene/ci/scripts/llvm-cache.py

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: MIT OR Apache-2.0
+# SPDX-FileCopyrightText: The Ferrocene Developers
+
+import argparse
+import subprocess
+import hashlib
+import os
+import io
+import shutil
+import pathlib
+import tarfile
+import sys
+
+CACHE_BUCKET="ferrocene-ci-caches"
+CACHE_PREFIX="prebuilt-llvm"
+
+
+TARBALL_PATH = "llvm-cache.tar"
```

Comment:
This was moved to a local folder because `/tmp/` is not very Windows friendly and in Python we are less covered by bashisms. We could use https://docs.python.org/3/library/tempfile.html if we wanted, but I don't really see much harm in this.